src/state/user/hooks.tsx

```diff
@@ -238,7 +238,10 @@ export function useUserAddedTokens(): Token[] {
 
   return useMemo(() => {
     if (!chainId) return []
-    return Object.values(serializedTokensMap?.[chainId] ?? {}).map(deserializeToken)
+    const tokenMap: Token[] = serializedTokensMap?.[chainId]
```

Comment:
nit: A cleaner solution may be to just type `Object.values<SerializedToken>(...)`